lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -185,3 +185,42 @@ class Toolbar(NavigationToolbar2Tk):
     print("success")
     Toolbar(fig.canvas, fig.canvas.manager.window)  # This should not raise.
     print("success")
+
+
+@pytest.mark.backend('TkAgg', skip_on_importerror=True)
+@_isolated_tk_test(success_count=2)
+def test_embedding():  # pragma: no cover
```

Comment:
Should get coverage now.
```suggestion
def test_embedding():
```

Reply:
Did something change about this notice?
```python
def _isolated_tk_test(success_count, func=None):
    """
    A decorator to run *func* in a subprocess and assert that it prints
    "success" *success_count* times and nothing on stderr.

    TkAgg tests seem to have interactions between tests, so isolate each test
    in a subprocess. See GH#18261

    The decorated function must be fully self-contained, and thus perform
    all the imports it needs.  Because its source is extracted and run by
    itself, coverage will consider it as not being run, so it should be marked
    with ``# pragma: no cover``
    """
```

Reply:
Ah sorry, #22005 is not merged yet.

Reply:
The tk parts of it moved to https://github.com/matplotlib/matplotlib/pull/22498 but still not merged.